src/main/java/software/aws/neptune/gremlin/GremlinConnectionProperties.java

```diff
@@ -1120,4 +1120,45 @@ protected void validateProperties() throws SQLException {
     public boolean isSupportedProperty(final String name) {
         return SUPPORTED_PROPERTIES_LIST.contains(name);
     }
+
+    /**
+     * Updates the SERVICE_REGION env variable in JVM to the region set by Gremlin connection properties
+     */
+    private void setServiceRegionEnvironmentVariable(final String region) throws SQLException {
+        if (region.equals(System.getenv("SERVICE_REGION"))) {
+            return;
+        }
+        if (System.getenv("SERVICE_REGION") != null) {
+            LOGGER.info(String.format("Overriding the current SERVICE_REGION environment variable with %s.", region));
+        }
+        try {
+            if (SystemUtils.IS_OS_WINDOWS) {
+                setWindowsEnvironmentVariable(region);
+            } else {
+                setMacEnvironmentVariable(region);
+            }
+        } catch (final Exception e) {
+            throw new SQLException(String.format("Error: unable to set SERVICE_REGION environment variable: %s", e));
```

Comment:
Nit: `Error: unable to set SERVICE_REGION environment variable: '%s'.`
Generally speaking, it's best to wrap %s String.format parameters with some sort of quote, in some cases of concatenated %s's, not having some sort of quote can lead to confusing outputs.